Sources_v3/Config/Token.swift

```diff
@@ -8,14 +8,24 @@ import Foundation
 public typealias Token = String
 
 /// A token provider is a function in which you send a request to your own backend to get a Stream Chat API token.
-/// Then you send it to the client to complete the setup with a callback function from the token provider.
+///
+/// Set your custom `TokenProvider` in `ChatClientConfig` when creating a `ChatClient` instance. `ChatClient` will use it whenever
+/// it needs to get a new token for the giver user.
```

Comment:
/// it needs to get a new token for the **given** user.

Reply:
fixed in 55e5b36392b4373a5d36d9b25c185888523fa9bd